cmd/launcher/launcher.go

```diff
@@ -363,17 +363,12 @@ func runLauncher(ctx context.Context, cancel func(), opts *launcher.Options) err
 		mirrorClient := http.DefaultClient
 		mirrorClient.Timeout = 5 * time.Minute // gives us extra time to avoid a timeout on download
 		tufAutoupdater, err := tuf.NewTufAutoupdater(
-			k.TufServerURL(),
+			k,
 			opts.RootDirectory,
```

Comment:
Do you want to pull `RootDirectory` from `k` as well?

Reply:
Thanks, yes, I thought it wasn't available there for some reason!